docs/secure-agent-communication.asciidoc

```diff
@@ -138,7 +138,8 @@ POST /_security/role/apm_agent_key_role
          "application":"apm",
          "privileges":[
             "event:write",
-            "config_agent:read"
+            "config_agent:read",
+            "sourcemap:write"
```

Comment:
This fixes https://www.elastic.co/guide/en/apm/guide/8.2/api-key.html#create-api-key-user by providing the `sourcemap:write` privilege to the created user so that the default selection in the APM UI works.